stakerservice/client/rpcclient.go

```diff
@@ -85,6 +86,26 @@ func (c *StakerServiceJSONRPCClient) Stake(
 	return result, nil
 }
 
+func (c *StakerServiceJSONRPCClient) BtcDelegationFromBtcStakingTx(
+	ctx context.Context,
+	stakerAddress string,
+	btcStkTxHash string,
+	globalParams *parser.GlobalParams,
```

Comment:
I actually like the idea of passing params in command, but maybe instead of passing whole thing lets pass `VersionedGlobalParams` i.e the version which should be used. 

It will be up to user to chose correct version, and it should simplify logic a bit.

Reply:
but then it might be difficult for the CLI user to pass the entire information (select the correct param and parse that into a file or something), so I will remove the params and expect to receive the data as primitive types